### DISCUSSION TITLE
Map each ordinal to a color

### DISCUSSION THREAD

**Discussion by casey** [ordinals/ord#796] [Ideas]
Each ordinal should map to a color. There are about 51 bits in an ordinal (log2 2099999997689999) so there are plenty of bits. I'd like to use the REC 2100 encoding, since it seems good.

First order of business is to come up with a suitable mapping from ordinal to color. I'm not sure if the color space should smoothly transition, so adjacent ordinals are close in color, or it should shuffle, so that adjacent ordinals are very different colors.

**Comment by loon3** [comment]
Love this idea.  Here's a quick proof of concept for creating a color palette for each ordinal by converting the ordinal number from decimal to hex.  The hex result is then broken into chunks of 6 characters each to be used as hex color codes.  If the final chunk is less than 6 characters, pad with zeros.  

https://codesandbox.io/s/serene-tu-l093mu?file=/src/App.js

Could also display as gradient instead of a palette or something else.
